crypto/merkle/bench_test.go

```diff
@@ -26,9 +26,10 @@ var innerHashTests = []*innerHashTest{
 func BenchmarkInnerHash(b *testing.B) {
 	b.ReportAllocs()
 
+	h := sha256.New()
```

Comment:
Why not `tmhash.New()` (just trying to understand the usage).
This also applies to other callsites of `sha256.New()` in this PR.